modular_skyrat/modules/ashwalkers/code/buildings/ash_tendril.dm

```diff
@@ -0,0 +1,164 @@
+		if(!istype(count_human.dna.species, /datum/species/lizard/ashwalker))
+			continue
+		allow_transform++
+
+	if(allow_transform < REQUIRED_OBSERVERS)
+		balloon_alert_to_viewers("[src] rejects the request, not enough viewers!")
+		playsound(src, 'sound/magic/demon_consume.ogg', 50, TRUE)
+		human_user.adjustBruteLoss(10)
+		return
+
+	else
+		balloon_alert_to_viewers("[src] reaches out to [human_user]...")
+		var/choice = tgui_alert(human_user, "Become an Ashwalker? You will abandon your previous life and body.", "Major Choice", list("Yes", "No"))
+
+		if(choice != "Yes")
+			balloon_alert_to_viewers("[src] feels rejected and punishes [human_user]!")
+			playsound(src, 'sound/magic/demon_consume.ogg', 50, TRUE)
+			human_user.adjustBruteLoss(50)
+			return
+
+		balloon_alert_to_viewers("[src] rejoices and transforms [human_user]!")
+		human_user.unequip_everything()
+		human_user.set_species(/datum/species/lizard/ashwalker)
+		human_user.underwear = "Nude"
+		human_user.update_body()
+		human_user.mind.add_antag_datum(/datum/antagonist/ashwalker)
+
+		if(SSmapping.level_trait(human_user.z, ZTRAIT_ICE_RUINS_UNDERGROUND) || SSmapping.level_trait(human_user.z, ZTRAIT_ICE_RUINS_UNDERGROUND))
+			ADD_TRAIT(human_user, TRAIT_NOBREATH, ROUNDSTART_TRAIT)
+			ADD_TRAIT(human_user, TRAIT_RESISTCOLD, ROUNDSTART_TRAIT)
+
+		ADD_TRAIT(human_user, TRAIT_PRIMITIVE, ROUNDSTART_TRAIT)
+		playsound(src, 'sound/magic/demon_dies.ogg', 50, TRUE)
+		meat_counter++
+
+	return ..()
+
+//this is the skyrat override
+/obj/structure/lavaland/ash_walker/consume()
+	for(var/mob/living/viewable_living in view(src, 1)) //Only for corpse right next to/on same tile
+		if(!viewable_living.stat)
+			continue
+
+		for(var/obj/item/sacrifice_posession in viewable_living)
+			if(!viewable_living.dropItemToGround(sacrifice_posession))
+				qdel(sacrifice_posession)
+
+		if(issilicon(viewable_living)) //no advantage to sacrificing borgs...
+			viewable_living.gib()
+			continue
+
+		if(viewable_living.mind?.has_antag_datum(/datum/antagonist/ashwalker) && (viewable_living.key || viewable_living.get_ghost(FALSE, TRUE))) //special interactions for dead lava lizards with ghosts attached
+			revive_ashwalker(viewable_living)
+			continue
+
+		if(ismegafauna(viewable_living))
+			meat_counter += MEGAFAUNA_MEAT_AMOUNT
+
+		else
+			meat_counter++
+
+		playsound(get_turf(src),'sound/magic/demon_consume.ogg', 100, TRUE)
+		var/delivery_key = viewable_living.fingerprintslast //key of whoever brought the body
+		var/mob/living/delivery_mob = get_mob_by_key(delivery_key) //mob of said key
+
+		//there is a 40% chance that the Lava Lizard unlocks their respawn with each sacrifice
+		if(delivery_mob && (delivery_mob.mind?.has_antag_datum(/datum/antagonist/ashwalker)) && (delivery_key in ashies.players_spawned) && (prob(40)))
+			to_chat(delivery_mob, span_warning("<b>The Necropolis is pleased with your sacrifice. You feel confident your existence after death is secure.</b>"))
```

Comment:
Don't we have a `span_boldwarning()` already?

Reply:
yes, but this is an upstream thing-- I will do it in this PR, but yea.